### DISCUSSION TITLE
How to identify queries coming from Prisma?

### DISCUSSION THREAD

**Discussion by bigethan** [prisma/prisma#13495] [Q&A] [answered]
I'm trying to see if there's a way to prefix/append a comment to all the queries that Prisma makes so that I can identify that they are coming from Prisma and not from a "generic" query. I don't see a way to do this in the docs, but also I might be missing something?  Any guidance would be appreciated.

**Comment by bigethan** [comment, marked as answer]
And I found an issue with this after tweaking my search terms, currently the asnwer is "you can't"  https://github.com/prisma/prisma/issues/10746

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
